### DISCUSSION TITLE
Colors in the lists depending on sorting key

### DISCUSSION THREAD

**Discussion by tiptruck** [TechbeeAT/jtxBoard#468] [Ideas]
Hi

Is there any discussion of setting colors in the background in the different lists?  For example, when I'm sorting by status then set the colors of the cards to show priorities and when I sort by priorities then set the colors to show the status.  Or would this just make things really messy and not help at all?

Regards Craig

**Comment by patrickunterwegs** [comment]
Hey @tiptruck , 
nope, I haven't had such a discussion. So let's discuss. ;-)
Basically there are some UI changes in v2.4.0, the colored edge was removed and the information in the top row is getting replaced by badges (that can have a background color). The item color will be represented as a border of the cards in the list. So this is where we go from. 

I think coloring the whole card would look a bit messy, that's why I avoided doing this in the first place already if you have a collection color or a color for the entry. 
v2.4.0 will also give you the option to define a color for categories and resources, the color will be the background color of the badge in the first row in the list. I'll attach a screenshot below for better understanding. I would suggest to also use different colors for different priorities for the background of the badge with the priority. What do you think? (I hope I was clear enough though ^^)

![Screenshot_1681635431](https://user-images.githubusercontent.com/72232737/232288060-f4548fcd-c03d-44f3-975d-a8a44210cfdb.png)

**Comment by tiptruck** [comment]
I agree that colouring the whole card would look messy unless the colour is very soft.  Attached is a screenshot of the widget for tasks.org.  They colour the completed box only, which looks good if you display the completed box.  For that to work then the completed box would have to sit to the left or it would be difficult to see.  I'm more thinking about the colours changing depending on the sort key

![Screenshot_20230416_163951_One UI Home](https://user-images.githubusercontent.com/53040514/232309439-680f53b4-a57c-4f6e-85fe-3837c2ce060c.jpg)
